tests/validation/converters/test_pandapower_converter_output.py

```diff
@@ -164,7 +263,7 @@ def test_attributes(output_data: Tuple[PandaPowerData, PandaPowerData], componen
     pd.testing.assert_series_equal(actual_values, expected_values, atol=5e-4, rtol=1e-4)
 
 
-@pytest.mark.parametrize(("component", "attribute"), component_attributes_df(load_and_convert_pgm_data_3ph()))
+@pytest.mark.parametrize(("component", "attribute"), component_attributes_df(load_and_convert_pgm_data_3ph())) 
```

Comment:
i will run the formatter for you once you've notified us that the current state is good.